hand_offs.md

```diff
@@ -0,0 +1,40 @@
+# Client Handoffs
+
+All Start
+1. Two weeks before or more!, write an email including PMs on both sides to set the date 
+and clarify any expectations around handoff at project end.
+1. Be sure you have a complete [readme](https://github.com/abtion/guidelines/blob/master/standard_readme.md) 
+incluing any diagrams or introductions to libraries that are heavily used as
+well as any known issues.
+
+Heroku apps
+1. Make sure all related Heroku plugins are pointed to client emails (ex:
+librato alerts or rollbar alerts)
+1. Make sure all PRs are closed and push to heroku, then transfer the pipeline 
+in the GUI. Client will setup their own github with what they pull from Heroku.
+
+
+PHP apps
```

Comment:
Why is this related to PHP only?

Reply:
Good question :) Going thought is that PHP apps don't use Heroku (or if they do, the clients we have do not want Heroku transferred). So, for our current suite of clients with php apps, we should shut down Heroku and transfer Github. For our rails clients, the opposite is true. They all want Heroku, and will create their own githubs.

We could also always handoff both...